mapie/multi_label_classification.py

```diff
@@ -781,13 +838,29 @@ def predict(
             len(alpha_np),
             axis=2
         )
-        self.r_hat, self.r_hat_plus = self._get_r_hat_plus(
-            bound,
-            delta,
-        )
-        self.lambdas_star = self._find_lambda_star(self.r_hat_plus, alpha_np)
-        y_pred_proba_array = (
-            y_pred_proba_array >
-            self.lambdas_star[np.newaxis, np.newaxis, :]
-        )
+        if self.metric_control == 'precision':
+            self.n_obs = len(self.risks)
+            self.r_hat = self.risks.mean(axis=0)
+            self.n_obs = len(self.risks)
+            self.valid_index, self.p_values = _ltt_procedure(self.r_hat,
+                                                             alpha_np,
+                                                             delta,
+                                                             self.n_obs)
+            self._check_valid_index()
+            self.lambdas_star, self.r_star = _find_lambda_control_star(
+               self.r_hat, self.valid_index, self.lambdas)
+
+            return y_pred, y_pred_proba_array
```

Comment:
UPD: this line exists in l866. It should be removed in consequence.

Reply:
Done

Reply:
Done